src/RpcServer/RpcServer.cs

```diff
@@ -232,16 +228,16 @@ private async Task<JObject> ProcessRequestAsync(HttpContext context, JObject req
             JToken @params = request["params"] ?? new JArray();
             if (!request.ContainsProperty("method") || @params is not JArray)
             {
-                return CreateErrorResponse(request["id"], -32600, "Invalid Request");
+                return CreateErrorResponse(request["id"], RpcError.InvalidRequest);
             }
             JObject response = CreateResponse(request["id"]);
             try
             {
                 string method = request["method"].AsString();
                 if (!CheckAuth(context) || settings.DisabledMethods.Contains(method))
-                    throw new RpcException(-400, "Access denied");
+                    throw new RpcException(RpcError.AccessDenied);
```

Comment:
It's not in the proposal, but we may include it: https://github.com/neo-project/proposals/pull/156#issuecomment-1822187290.

Reply:
In general, -400 is replaced by -302 as mentioned in neo-project/proposals#156, but I guess we've missed this meaning of it in #728 and reusing -302 would be counterproductive. I'd use 6XX, it's related to server configuration.

Reply:
Please check https://github.com/neo-project/proposals/pull/156/commits/39787232cc072d086318ec78f48cabdcbbc565b5

Reply:
Good to go, let's use `-600` then.